runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/ai.ts

```diff
@@ -0,0 +1,10 @@
+const aiTranslations = {
+    'Process.Details.Modeler.Actions.Ai.Chat.Label': 'Chat',
+    'Process.Details.Modeler.Actions.Ai.Chat.Input.Message': 'Message',
+    'Process.Details.Modeler.Actions.Ai.Chat.Input.ImagePath': 'Image path',
+    'Process.Details.Modeler.Actions.Ai.Chat.Input.Message.Info': 'Message for chat, which will interpreted in order to give a response',
```

Comment:
Message for chat to interpret OR will be interpreted. And tbh I would leave just this, "in order to give a resposne" is imo too much :D same in Polish, but if you think it is helpful - leave it (but maybe change "in order to get a response"